### ISSUE TITLE
Configurable merge conflict message

### ISSUE THREAD

**Issue by marcoadkins** [chinthakagodawita/autoupdate#101]
Thoughts on adding a configuration value to support custom merge conflict messages similar to the existing `MERGE_MSG` setting?

**Comment by chinthakagodawita** [comment]
Hey @marcoadkins - where would this message be displayed? In the log or as a comment in the PR?

`MERGE_MSG` is a little different as it controls the commit message. autoupdate makes no changes to the branch if it detects a conflict since conflicts require a manual resolution.

**Comment by marcoadkins** [comment]
@chinthakagodawita I was thinking as a comment on the PR. To alert authors that they have a conflict to resolve.

**Comment by chinthakagodawita** [comment]
Yep, that's definitely on the cards (see #11). I can't promise an ETA just yet though but happy to accept PRs if you feel like giving it a crack 🙂

**Comment by marcoadkins** [comment]
I see. I started working on this a day or so ago. It's a little more specific to my use case but could easily pull out the jira logic and just leave the commenting strategy there. So maybe once I am done with what I need I will go back and PR just the comment side. 

https://github.com/chinthakagodawita/autoupdate/compare/master...marcoadkins:master

**Comment by chinthakagodawita** [comment]
Oh nice one, looking forward to it!

**Comment by marcoadkins** [comment]
@chinthakagodawita Hit a bit of a snag with the issue comment feature. Ended up spamming our repos every time a push happened after a conflict existed. So need to identify a reliable way to prevent duplicate comments on the same conflict. I am think either a label that gets added/removed when conflicts arise/subside. Or would need to fetch the issue comments and compare the git hash they were added on if thats possible...

**Comment by chinthakagodawita** [comment]
Yea I was worried about that.

One option is to include a hidden HTML tag in the comment, load existing comments, check if any of them have that hidden tag and leave a conflict message comment if not.

Here's an example of another GH action that does that: https://github.com/marocchino/sticky-pull-request-comment/blob/17dbcbf6a4e62aa43b811fc51f40c4f5b1445f78/src/comment.ts#L7
